Binance.Net/SubClients/BinanceClientSubAccounts.cs

```diff
@@ -21,11 +21,11 @@ namespace Binance.Net.SubClients
     /// </summary>
     public class BinanceClientSubAccount : IBinanceClientSubAccount
     {
-        private const string subAccountListEndpoint = "sub-account/list.html";
-        private const string subAccountTransferHistoryEndpoint = "sub-account/transfer/history.html";
+        private const string subAccountListEndpoint = "sub-account/list";
+        private const string subAccountTransferHistoryEndpoint = "sub-account/sub/transfer/history";
         private const string transferSubAccountEndpoint = "sub-account/transfer.html";
```

Comment:
```suggestion
        private const string transferSubAccountEndpoint = "sub-account/universalTransfer";
```

Reply:
I asked JKorf about this one. They added 2 Endpoints, GET and POST for /sapi/v1/sub-account/universalTransfer

Reply:
POST requests will do something when executed, GET requests only retrieve data. It's very common that the same url has both a GET and a POST endpoint, with different functionality. The `_baseClient.SendRequestInternal` has a parameter specifying if it is a GET or POST request. 
There should be 2 methods, 1 for the GET, which retrieves the transfer history, and 1 for the POST which creates a new transfer